src/deviceApiCalls/transports/apple.transport.js

```diff
@@ -14,6 +15,14 @@ export class AppleTransport extends DeviceApiTransport {
             secret: this.config.secret,
             hasModernWebkitAPI: this.config.hasModernWebkitAPI
         }
+        if (!this.sendOptions.hasModernWebkitAPI) {
+            // @ts-ignore
```

Comment:
Is this ignore still needed or a leftover?

Reply:
for now still needed - we can have a follow up to improve the types though, for sure :)